bundles/org.openhab.binding.haassohnpelletstove/README.md

```diff
@@ -0,0 +1,67 @@
+# Haas Sohn Pellet Stove Binding
+
+The binding for Haassohnpelletstove communicates with a Haas and Sohn Pelletstove through the optional
+WIFI module. More information about the WIFI module can be found here: https://www.haassohn.com/de/ihr-plus/WLAN-Funktion
+
+## Supported Things
+
+| Things | Description  | Thing Type |
+|--------|--------------|------------|
+| haassohnpelletstove | Control of a Haas & Sohn Pellet Stove| oven|
+
+
+## Thing Configuration
+
+In general two parameters are required. The IP-Address of the WIFI-Modul of the Stove in the local Network and the Access PIN of the Stove.
+The PIN can be found directly at the stove under the Menue/Network/WLAN-PIN
+
+```
+Thing haassohnpelletstove:oven:myOven "Pelletstove"  [ hostIP="192.168.0.23", hostPIN="1234"]
+```
+
+## Channels
+
+The following channels are yet supported:
+
+
+| Channel | Type  | Access| Description|
+|---------|-------|-------|------------|
+| power| Switch | read/write|Turn the stove on/off|
+|channelIsTemp|Number:Temperature|read|Receives the actual temperature of the stove|
+|channelSpTemp|Number:Temperature|read/write|Receives and sets the target temperature of the stove|
+|channelMode|String|read|Receives the actual mode the stove is in like heating, cooling, error, ....|
+|channelEcoMode|Switch|read/write|Turn the eco mode of the stove on/off|
+|channelIngitions|Number|read|Amount of ignitions of the stove|
+|channelMaintenanceIn|Number:Mass|read|States the next maintenance in kg|
+|channelCleaningIn|String|read|States the next cleaning window in hours:minutes as string|
+|channelConsumption|Number:Mass|read|Total consumption of the stove|
+|channelOnTime|Number|read|Operation hours of the stove|
+
+## Full Example
+
+demo.items:
+
+```
+Number:Temperature isTemp { channel="oven:channelIsTemp" }
+Number:Temperature spTemp { channel="oven:channelSpTemp" }
+String mode { channel="oven:channelMode" }
+Switch power { channel="oven:power" }
+```
+
+## Google Assistant configuation
+See also: https://www.openhab.org/docs/ecosystem/google-assistant/
```

Comment:
There must be an empty line between heading and text in markdown.

Reply:
ok